docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>"If user" -&gt; If a user</p><p>"e.g., file name" -&gt; e.g., a file name</p><p> </p><p>I would explain better where the target automatically generated by RP will transfer an input file. In that explanation, I would also add an example. I would propose text for that but I myself do not know how that automatic generation works. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I agree with Matteo; an example with multiple actions will be helpful for the reader.</p>

Reply:
<p>resolved</p>